### DISCUSSION TITLE
Not-found.jsx file, issue with window.innerWidth

### DISCUSSION THREAD

**Discussion by EgzonBajrami** [vercel/next.js#51362] [Help] [answered]
### Summary

I have a checker for when an user is in mobile viewport so I can conditionally render the header.
The issue is this works for every page but not for not-found.jsx file, which calls the same component as other pages.

const isMobile = typeof window !== "undefined" && window.innerWidth <= 600

this is what I'm using to check when an user is in mobile, note that this does work on every other page.

Also I would like to point this out, it seems like in general this page is zoomed out and shows incorrect pixel ratio in mobile devices, like after it reaches 600pxs width it just fails to even detect the media queries for under that ratio.

So far what I've tried, rendering the whole thing on client doesn't work, wrapping everything into another component and then sending it to the not-found.jsx doesn't work. Another issue I ran into is Link not working  on development but on production it works.

I've tested this out in both production and development, doesn't work on neither.

**Comment by icyJoseph** [comment]
What next.js version? I wonder if the `<meta name="viewport" content="width=device-width, initial-scale=1.0">` is missing somehow.

**Reply by icyJoseph** [reply]
I just saw this on my own site... gonna investigate

**Reply by icyJoseph** [reply, marked as answer]
![Screenshot 2023-06-15 at 19 00 14](https://github.com/vercel/next.js/assets/21013447/2df37eb8-0958-46b6-b294-6deb40623cb4)

For reference, the meta tag for viewport is missing.

**Reply by icyJoseph** [reply]
Here you go: https://github.com/vercel/next.js/pull/50044

Fixed in next@13.4.5

**Comment by EgzonBajrami** [comment]
> What next.js version? I wonder if the `<meta name="viewport" content="width=device-width, initial-scale=1.0">` is missing somehow.

I'm using 13.4.3 and yeah, actually all of my layout items are missing, missed the meta tags when checking site, missed title but I added it in the not-found file. Thanks for your answer.
